sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_multiple_node/language-modeling.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Add another variable for n_gpu.</p><p> </p><p>Might be better to use <code>ml.p3.16xlarge</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Also your comment (  <code># ml.p3.8xlarge is easily available. However, ml.p3.16xlarge provides better performance.</code>  ) does not fit with the use of ml.p3dn.24xlarge. I agree it's better to use ml.p3.16xlarge, and keep comment as-is. If you use the 24xlarge, revise the comment.</p>